server/graphql/requisition_line/src/lib.rs

```diff
@@ -53,4 +53,13 @@ impl RequisitionLineMutations {
     ) -> Result<response_requisition_line::UpdateResponse> {
         response_requisition_line::update(ctx, &store_id, input)
     }
+
+    async fn delete_response_requisition_line(
+        &self,
+        ctx: &Context<'_>,
+        store_id: String,
+        input: response_requisition_line::DeleteInput,
+    ) -> Result<response_requisition_line::DeleteResponse> {
+        response_requisition_line::delete(ctx, &store_id, input)
+    }
```

Comment:
I've made this as per delete_request_requisition_line, but neither are currently exposed

Reply:
It's being used by the batch mutation. Ok not to be exposed

Reply:
of course!